pyiron/base/job/interactive.py

```diff
@@ -263,6 +265,66 @@ def interactive_open(self):
         """
         self.server.run_mode.interactive = True
 
+    def hd_copy(self, hdf_old, hdf_new, exclude_groups = []):
+        """
+        args:
+            hdf_old (ProjectHDFio): old hdf
+            hdf_new (ProjectHDFio): new hdf
+            exclude_groups (list): list of groups to delete
+        """
+        for p in hdf_old.list_nodes():
+            hdf_new[p] = hdf_old[p]
+        for p in hdf_old.list_groups():
+            if p in exclude_groups:
+                continue
+            h_new = hdf_new.create_group(p)
+            self.hd_copy(hdf_old[p], h_new, exclude_groups=exclude_groups)
+        return hdf_new
+
+    def file_size(self, hdf):
+        """
+        args:
+            hdf (ProjectHDFio): hdf file
+        """
+        return os.path.getsize(hdf.file_name)
+
+    def get_size(self, hdf):
```

Comment:
Can not this be implemented in pyiron.base.generic.hdfio.ProjectHDFio?

Reply:
ok I'll try